docs/game_data/spel2.lua

```diff
@@ -6320,9 +6335,9 @@ function CustomTheme:new() end
 Vec2 = nil
 ---@return Vec2
 function Vec2:new() end
----@param vec2 Vec2
+---@param 2 Vec
 ---@return Vec2
-function Vec2:new(vec2) end
+function Vec2:new(2) end
```

Comment:
Broken for some reason

Reply:
Well yeah spel2.lua as a whole just seems worse now with these glitches, can't really merge that.